project/Dependencies.scala

```diff
@@ -126,10 +128,17 @@ object Dependencies {
   val googleCloudNio: ModuleID =  "com.google.cloud"    % "google-cloud-nio"      % "0.124.21" % Test // brought in for FakeStorageInterpreter
 
   val circeYaml =         "io.circe"          %% "circe-yaml"           % "0.14.2"
+  val circeCore =         "io.circe"          %% "circe-core"           % "0.14.3"
```

Comment:
this is brought in transitively. we don't need to specify here

Reply:
You're correct, the `circe-core` will be brought in transitively. Removed.